components/layout/setup-overview.tsx

```diff
@@ -66,7 +66,7 @@ const OverviewTemplate = (props) => {
           // images: [openGraphImage(frontmatter.title, '| TinaCMS Docs')],
         }}
       />
-      <DocsLayout navItems={props.oldNavDocs.data}>
```

Comment:
Do we also remove all instances of oldNavDocs var?

Reply:
Yep